java-frontend/src/main/java/org/sonar/java/AbstractJavaClasspath.java

```diff
@@ -51,7 +51,7 @@
 public abstract class AbstractJavaClasspath implements BatchExtension {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "BatchExtension"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)